composer.json

```diff
@@ -12,6 +12,7 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
+        "ext-json": "^8.0",
```

Comment:
Can you revert this change? As you can see from the GA, it's causing composer to fail.

Reply:
It could be `"ext-json": "*",`

Reply:
It was like that already.

Reply:
https://github.com/nunomaduro/larastan/blob/master/composer.json#L26

😲

Reply:
Ah yes I thought it only changed the order. I'll revert it